ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -619,6 +602,25 @@ svi_profiles:
         tags: [ < tag_1 >, < tag_2 > ]
 ```
 
+### SVI Profiles
+
+```yaml
+# Optional profiles to share common settings for SVIs
+# Keys are the same used under SVIs. Keys defined under SVIs take precedence.
+# Note, structured configuration is not merged recursilvy and will be taken directly from the most specific level in the following order:
```

Comment:
Maybe I missed some discussion here but why not merging recursively structured_config? In the case where I want to apply a profile with some structured_config on a node + adding some node_specific config it would mean that I need to copy all the profile structured_config back in host_vars right?

Reply:
Correct, this is because it is the current implementation - and this change needs to be non-breaking.
We would like to change this strategy in 4.0.

Reply:
Ack - resolving this